### DISCUSSION TITLE
How can I configure the version of python to use

### DISCUSSION THREAD

**Discussion by jesuscuevasnetkia** [Tecnativa/doodba#552] [Q&A] [answered]
Hello!!, We are using doddba to create a v16 environment and we wanted to use the 'rest-framework' fastapi module, this module uses 'Annotated' typing which is introduced in python version 3.9 and we have seen that we are using 3.8, is there a way to choose or change the version of python that we use?
Thank you so much !

**Comment by ap-wtioit** [comment, marked as answer]
You would need to build the image yourself and change the first line from
```
python:3.8-slim-bullseye AS base
```
to
```
python:3.9-slim-bullseye AS base
```
in
https://github.com/Tecnativa/doodba/blob/master/16.0.Dockerfile
and then of course test that everything is still working as expected

**Comment by jesuscuevasnetkia** [comment]
Thank you very much @ap-wtioit , it has worked perfectly for me
